online-bookstore/src/main/java/group1/itschool/onlinebookstore/models/dto/BookDTO.java

```diff
@@ -5,6 +5,20 @@
 @Data
```

Comment:
Review the code and submit another commit from your side with all the proper validations on the fields.